src/async_impl/client.rs

```diff
@@ -53,7 +53,7 @@ use crate::{IntoUrl, Method, Proxy, StatusCode, Url};
 /// The `Client` holds a connection pool internally, so it is advised that
 /// you create one and **reuse** it.
 ///
-/// You do **not** have to wrap the `Client` it in an [`Rc`] or [`Arc`] to **reuse** it,
+/// You do **not** have to wrap the `Client` in an [`Rc`] or [`Arc`] to **reuse** it,
 /// because it already uses an [`Arc`] internally.
 ///
 /// [`Rc`]: std::rc::Rc
```

Comment:
I'm also not sure if we need this anymore since [`Rc`] will automatically link to the docs on newer Rust versions. I see we didn't have an explicit link for [`Arc`] so I wasn't totally sure what to do here.